app/tag-management/edit-tag/edit-tag.component.css

```diff
@@ -0,0 +1,104 @@
+h3 {
```

Comment:
do not alter headlines. We want to be consistent on all our sub-sites, so headlines are defined in the overall css file `hip-theme`